doc/changes.rst

```diff
@@ -5,7 +5,7 @@ Change Log
 1.9.10 (unreleased)
 -------------------
 
-* No changes yet.
+* Add bspline to funcfits.  scipy is now a dependency
```

Comment:
I have mixed feelings about adding scipy as an explicit dependency.  Will `bspline_fit()` be used by multiple DESI packages?  Because if it is being used by only one package, then it should be added to that package, not desiutil.